Description: Allow any pytz version
 Upstream requires a specific version of pytz. We don't want this restriction
 in Debian, the normal pytz package should be ok (it's heavily patched anyway).
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2015-09-04

--- python-oslo.utils-2.1.0.orig/requirements.txt
+++ python-oslo.utils-2.1.0/requirements.txt
@@ -13,7 +13,7 @@ six>=1.9.0
 iso8601>=0.1.9
 oslo.i18n>=1.5.0 # Apache-2.0
 monotonic>=0.1 # Apache-2.0
-pytz>=2013.6
+pytz
 netaddr>=0.7.12
 netifaces>=0.10.4
 debtcollector>=0.3.0 # Apache-2.0
